pkg/kinflate/tree/builder.go

```diff
@@ -64,30 +68,38 @@ func manifestToManifestData(m *manifest.Manifest) (*ManifestData, error) {
 }
 
 // loadManifestFileFromPath loads a manifest object from file.
-func loadManifestFileFromPath(filename string) (*manifest.Manifest, error) {
-	loader := kutil.ManifestLoader{fs.MakeRealFS()}
-	m, err := loader.Read(filename)
+func (l *Loader) loadManifestFileFromPath(filename string) (*manifest.Manifest, error) {
+	var loader = kutil.ManifestLoader{FS: l.FS}
+	fullFilename, err := loader.MakeValidManifestPath(filename)
+	if err != nil {
+		return nil, err
+	}
+	l.Path = fullFilename
+	m, err := loader.Read(fullFilename)
 	if err != nil {
 		return nil, err
 	}
+	loader.Validate(m)
```

Comment:
I know this function is a placeholder for now, but shouldn't it return an error and have the error handled?

Reply:
I will move this change to a different PR and update it there.